From edacd0568487815ca3a0821c25912133cb6c1731 Mon Sep 17 00:00:00 2001
From: Han Xu <b45815@freescale.com>
Date: Thu, 8 Oct 2015 18:16:02 -0500
Subject: [PATCH] MLK-11573: ARM: dts: fix the iomux pin conflict for LPSR
 mode

add the pinctrl-1 for iomux in ecspi device tree to avoid conflict.

Signed-off-by: Han Xu <b45815@freescale.com>
---
 arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-ecspi.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-ecspi.dts b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-ecspi.dts
index b74863f..16dd447 100644
--- a/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-ecspi.dts
+++ b/arch/arm/boot/dts/imx7d-12x12-lpddr3-arm2-ecspi.dts
@@ -23,4 +23,5 @@
  */
 &iomuxc {
 	pinctrl-0 = <&pinctrl_hog_1>;
+	pinctrl-1 = <&pinctrl_hog_1>;
 };
-- 
1.8.0

